src/Core/Grid/Definition/GridDefinitionInterface.php

```diff
@@ -75,4 +75,12 @@ public function getGridActions();
      * @return FilterCollectionInterface
      */
     public function getFilters();
+
+    /**
+     * Specify an uuid to match a filter, it is also used as a scope for
+     * this grid request parameters.
+     *
+     * @return string
+     */
+    public function getFiltersUuid();
```

Comment:
In theory, thi should be considered as BC?

Reply:
Yes 😢 let's wait for the end of this PR (this is still a WIP) to see how we can handle this

Reply:
Yeah I didn't think about this at first, my bad
The thing is I need to identify which grid use the new filters unique key (or uuid) system.. I need this to detect how to format parameters in the request so that each grid has its own "scope":
- `?limit=10` (for former grids)
- `?language[limit]=10` (for new grids)

It would be much simpler if ALL grids worked the same way, this way I could simply use the gridId everywhere.

But I also would like to be able to define this "scope" later, this would allow to have multiple grids of the same entity using the same filter classes on the same page, each one defining its request scope and filter unique id.

(Of course this will not be managed fully automatically by the resolver, but you could use the new `FiltersBuilder` service with just a few settings to override the generic behavior - just change scope and filters uuid)